src/components/OrderSummary/OrderCard.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Grid, Divider, Box } from '@mui/material';
+import { SharedImage } from '../shared/Image/SharedImage';
+import SharedTypography from '../shared/Text/SharedText';
+import {
+  OrderCardContainer,
+  ItemCard,
+  ImageStyle,
+  SharedFlexBox,
+  dividerStyle,
+} from './OrderCard.styled';
+import SharedGrid from '../shared/Grid/SharedGrid';
+import { UI_TEXT } from '../../constants/text';
+import { convertToShekels } from '../../utils/converting';
+
+const OrderCard = ({ items = [], total = 0 }) => {
+  return (
+    <OrderCardContainer>
+      {items.length === 0 ? (
+        <SharedTypography variant="body1">
+          {UI_TEXT.NO_ITEMS_IN_CART}
+        </SharedTypography>
+      ) : (
+        <>
+          <SharedGrid container direction="column">
+            {items.map((item, index) => (
+              <Grid item key={index}>
+                <ItemCard>
+                  <SharedFlexBox>
+                    <SharedImage
+                      src={item.image}
+                      alt={item.name}
+                      style={ImageStyle}
+                    />
+                  </SharedFlexBox>
+                  <SharedTypography variant="body1">
+                    {item.name}
+                  </SharedTypography>
+                  <SharedTypography variant="body2" fontWeight="medium">
+                    {convertToShekels(item.price)}
+                  </SharedTypography>
```

Comment:
Can you do something like this : 

```js
const shekelPrice = convertToShekels(item.price);
```

And use the price instead of calling the function every time? 
Because this function will get called in every render no matter what